### ISSUE TITLE
Output IPMI Sol to noVNC

### ISSUE THREAD

**Issue by LeandroTakagi** [novnc/noVNC#1028]
Hi
I'm trying to output ipmi sol connection using noVNC.
I tried ```./util.launch.sh --vnc [IPMI ip]:623``` but unsuccessful. I tried using port 17990 too, but not connect.
I tried using qemu too, but unsuccessful.

If anyone has tried this, can help me?
Thanks

**Comment by DirectXMan12** [comment]
We've had some success with management boards and such that use iKVM (see #385 and friends), but without knowing more about your specific setup, I'm afraid it'll be difficult to help out.

**Comment by LeandroTakagi** [comment]
Thank you for answering.
Explaining more my problem:

I have an server with iLO and IPMI access.
I want to create a web console with this server using ipmi sol (serial over lan). The solution I think is redirect sol connection (ipmitool sol activate) to a kvm device.

Searching in DigitalOcean console, I see they are using noVNC to create the console, and searching more I got a way to connect an Ubuntu console using qemu (```qemu-system-i386 -vnc :1 -nographic -serial mon:stdio -append 'console=ttyS0' binary.img -kernel /boot/vmlinuz-3.13.0-141-generic```) and output it using noVNC (```./utils/launch.sh --vnc localhost:5901```).

So now I need to remotelly redirect ipmi sol to access this server console, but I don't have idea in how to create this redirection, and if it is possible.
I have tried an other solution using iLO configured ports to connect it direct to noVNC, but not worked and got Connection refused.
